### ISSUE TITLE
LDAP: new users doesn't have mailbox

### ISSUE THREAD

**Issue by josph2001** [modoboa/modoboa#1471]
# Impacted versions

* Modoboa: 1.10.4
* installer used: Yes
* Webserver: Nginx

# Steps to reproduce
- Upgraded an old version (1.9) to current version.
- LDAP authentication only
- Create a new entry in LDAP then login modoboa.

# Current behavior
- New user can access only to Settings page, but not webmail. It shows error "Permission denied: a mailbox is required". I'm trying to enable DEBUG mode however there is no additional information.
- Existing users can use Webmail normally.
- I'm trying to investigate more into the issue. Here are other information
   - New user is not listed in admin panel
   - There is an entry for the new user in `core_user` table but not in `admin_mailbox`.
   - Email client (thunderbird) is working properly still. 

# Expected behavior
- New ldap user should be able to use webmail after first logging in.

# Video/Screenshot link (optional)

**Comment by josph2001** [comment]
@tonioo The option is already enabled. I didn't try it yet since the beginning.

<img width="484" alt="screen shot 2018-04-19 at 3 28 33 pm" src="https://user-images.githubusercontent.com/26566869/38980204-6749068a-43e6-11e8-983c-596b15ea8817.png">

**Comment by tonioo** [comment]
What is your LDAP directory?

**Comment by josph2001** [comment]
I'm using OpenLDAP.

**Comment by tonioo** [comment]
@josph2001 What username is used to login?

**Comment by josph2001** [comment]
@tonioo the email address is being used to login. It will be matched with the email field in LDAP user.

**Comment by josph2001** [comment]
@tonioo Is it what you are mentioning about?

![ldap_config](https://user-images.githubusercontent.com/26566869/39355849-8c11538e-4a38-11e8-9b2e-79df90aa439f.jpg)

**Comment by tonioo** [comment]
@josph2001 Have you seen this issue with more than 1 user? Looks like something failed during the creation process...

**Comment by josph2001** [comment]
@tonioo yes, I had tried with few users in same domain or in different one.

**Comment by josph2001** [comment]
@tonioo as far as my observation, if I create a mailbox first from admin panel, then login with ldap credentials, everything will be fine. I guess auto mailbox creation is not working properly.

**Comment by tonioo** [comment]
@josph2001 I really don't understand what is happening... Could it be possible to access your server?

**Comment by josph2001** [comment]
@tonioo it seems the issue was with ldap lib (django-auth-ldap), I was upgraded to 1.6.2 (from 1.2) then everything is fine now. Thanks!
